package/MDAnalysis/analysis/encore/__init__.py

```diff
@@ -0,0 +1,22 @@
+# __init__.py
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from Ensemble import *
```

Comment:
I think it is OK in that specific instance assuming that:

* it is to give direct access to methods and attributes of a module (it is in `__init__.py`)
* it is restrained to imports within the module
* all the submodules that are imported this way define the `__all__` attribute.

Of course, this is nothing but mu opinion.